diff -up mpv-0.29.0/player/main.c.1~ mpv-0.29.0/player/main.c
--- mpv-0.29.0/player/main.c.1~	2018-09-01 02:23:25.263607469 +0200
+++ mpv-0.29.0/player/main.c	2018-09-01 02:26:39.289397574 +0200
@@ -387,12 +387,15 @@ int mp_initialize(struct MPContext *mpct
         // Distro maintainers who patch this out should be aware that mpv
         // intentionally ignores ABI in some places where it's not possible to
         // get by without violating it.
-        print_libav_versions(mpctx->log, MSGL_FATAL);
+	// Yet it's almost always safe to update e.g. ffmpeg 3.2.0 to 3.2.1 to
+	// 3.2.2 without rebuilding everything. Let's not overreact, but let's
+	// issue a warning so people know what's up if something goes wrong.
+        print_libav_versions(mpctx->log, MSGL_WARN);
         MP_FATAL(mpctx, "\nmpv was compiled against a different version of "
-                 "FFmpeg/Libav than the shared\nlibrary it is linked against. "
-                 "This is most likely a broken build and could\nresult in "
-                 "misbehavior and crashes.\n\nmpv does not support this "
-                 "configuration and will not run - rebuild mpv instead.\n");
+                 "FFmpeg/Libav than the shared\nlibrary it is linked against.\n"
+                 "This could\nresult in misbehavior and crashes.\n\n"
+		 "If you are seeing any of those, please rebuild mpv\n"
+		 "(Or ask #openmandriva-cooker to rebuild it for you).\n");
         return -1;
     }
 
