diff -up mpv-0.37.0/audio/decode/ad_spdif.c.omv~ mpv-0.37.0/audio/decode/ad_spdif.c
--- mpv-0.37.0/audio/decode/ad_spdif.c.omv~	2024-04-05 21:35:38.960439612 +0200
+++ mpv-0.37.0/audio/decode/ad_spdif.c	2024-04-05 21:35:44.456494421 +0200
@@ -53,7 +53,7 @@ struct spdifContext {
     struct mp_decoder public;
 };
 
-static int write_packet(void *p, uint8_t *buf, int buf_size)
+static int write_packet(void *p, const uint8_t *buf, int buf_size)
 {
     struct spdifContext *ctx = p;
 
diff -up mpv-0.37.0/filters/f_lavfi.c.omv~ mpv-0.37.0/filters/f_lavfi.c
--- mpv-0.37.0/filters/f_lavfi.c.omv~	2024-04-05 21:36:06.104710574 +0200
+++ mpv-0.37.0/filters/f_lavfi.c	2024-04-05 21:36:14.192791439 +0200
@@ -1029,7 +1029,7 @@ static const char *get_avopt_type_name(e
     case AV_OPT_TYPE_VIDEO_RATE:        return "fps";
     case AV_OPT_TYPE_DURATION:          return "duration";
     case AV_OPT_TYPE_COLOR:             return "color";
-    case AV_OPT_TYPE_CHANNEL_LAYOUT:    return "channellayout";
+    case AV_OPT_TYPE_CHLAYOUT:          return "channellayout";
     case AV_OPT_TYPE_BOOL:              return "bool";
     case AV_OPT_TYPE_CONST: // fallthrough
     default:
