--- mpv-0.6.0/etc/example.conf.0000~	2014-09-30 20:52:47.000000000 +0200
+++ mpv-0.6.0/etc/example.conf	2014-10-01 02:36:49.386782107 +0200
@@ -1,7 +1,7 @@
 #
 # mpv configuration file
 #
-# Configuration files are read system-wide from /usr/local/etc/mpv.conf
+# Configuration files are read system-wide from /etc/mpv.conf
 # and per-user from ~/.config/mpv/mpv.conf, where per-user settings override
 # system-wide settings, all of which are overridden by the command line.
 #
@@ -28,6 +28,7 @@
 
 # Specify default video driver (see --vo=help for a list).
 #vo=vdpau
+vo=opengl
 
 # Start in fullscreen mode by default.
 #fs=yes
@@ -50,6 +51,7 @@
 
 # Specify default audio driver (see --ao=help for a list).
 #ao=alsa
+ao=pulse,alsa,
 
 # Disable softvol usage, and always use the system mixer if available.
 #softvol=no
